infracheck/infracheck/model.py

```diff
@@ -12,21 +12,24 @@ class ExecutedCheckResult(object):
     hooks_output: str
     configured_name: str
     refresh_time: datetime
+    description: str
 
-    def __init__(self, configured_name: str, output: str, exit_status: bool, hooks_output: str):
+    def __init__(self, configured_name: str, output: str, exit_status: bool, hooks_output: str, description: str):
         self.configured_name = configured_name
         self.output = output
         self.exit_status = exit_status
         self.hooks_output = hooks_output
         self.refresh_time = datetime.now()
+        self.description = description
 
     @classmethod
-    def from_not_ready(cls, configured_name: str):
+    def from_not_ready(cls, configured_name: str, description: str):
         check = cls(
```

Comment:
Where is `check` used?

Reply:
You're right, I had to click once more. It was collapsed and I saw `return` from `to_hash()` instead :)